src/frame.c

```diff
@@ -24,9 +24,10 @@
 #define PCI_AUDIO_FIXED 0xE3634C
 #define PCI_AUDIO_FIXED_OPP 0x8D8D33
 
-#define BBM 0x42E23A7D
 #define MAX_AUDIO_PACKETS 64
 
+static uint8_t bbm[] = { 0x7D, 0x3A, 0xE2, 0x42 };
```

Comment:
Add a const. Also you may want to consider moving this into process_fixed_data as the first line since it is only used in that function.

Reply:
Done.